src/components/StaticMetadata/StaticMetadata.stories.mdx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs'
```

Comment:
Please convert this to .tsx so that we can enjoy type safety in storybook as well.